### ISSUE TITLE
make bytes available on ios

### ISSUE THREAD

**Issue by mesqueeb** [capawesome-team/capacitor-cloudinary#17]
since the cloudinary upload result will return `bytes` as well, we can use that so it's available on all platforms

**Comment by robingenz** [comment]
`bytes` should already by available on Android and Web, see https://github.com/capawesome-team/capacitor-cloudinary#uploadresourceresult. 

> The number of bytes of the uploaded resource. Only available on Android and Web.

Only the Cloudinary iOS SDK does not seem to have a simple bytes property:
![grafik](https://user-images.githubusercontent.com/13857929/226174882-0c60ed19-b793-4988-aebf-5ef63a6f20d2.png)

Since the property is already available on the Web, I am closing this issue. Let me know if there are any problems.

**Comment by robingenz** [comment]
I created a feature request for the Cloudinary iOS SDK: https://github.com/cloudinary/cloudinary_ios/issues/399

**Comment by mesqueeb** [comment]
Yes! I miswrote the issue! But support for iOS would be nice. Though we can get it through the file picker as well so not a huge priority for us.

**Comment by robingenz** [comment]
Alright, this should already be possible with the File Picker plugin, see `size` property in https://github.com/capawesome-team/capacitor-file-picker#file

**Comment by mesqueeb** [comment]
renamed title to `iOS`
